docs/Developers/Git_Basics.rst

```diff
@@ -89,6 +112,15 @@ will work.
 
     ``-b`` will create a new branch if the branch doesn't already exist.
 
+    Some sample naming conventions for branches:
+    + issueXXX
+    + patchXXX
+    + gh-XXX.
+    (Where XXX is your issue number.)
```

Comment:
please add "short form of issue name" as a possible naming convention. IMO that's the best one as the reviewer will get an idea of what the PR is about without even having to open the issue.